File: py/3.control_statement.py
# if, elif, else
n = int(input())

if n == 0:
    print("is zero")
elif 0 < n:
    print("is positive")
else:
    print("is negative")

# for
f = int(input())

for i in range(f):
    print(i, end=" ")
print()

# while
w = int(input())

while 0 < w:
    w -= 1
    print(w, end=" ")


'''
when input is 0, print SSCC once
else, print SSCC as absolute value of input
'''

File: cpp/3.control_statement.cpp
#include <stdio.h>

int main(){
    // if, else
    int n;
    scanf("%d", &n);

    if(n == 0){
        printf("is zero\n");
    }
    else if(0 < n){
        printf("is positive\n");
    }
    else{
        printf("is negative\n");
    }

    // for
    int f;
    scanf("%d", &f);

    for(int i = 0; i < f; i++)
        printf("%d ", i);
    printf("\n");

    // while
    int w;
    scanf("%d", &w);

    while(w--)
        printf("%d ", w);



    return 0;
}

/*
 * when input is 0, print SSCC once
 * else, print SSCC as absolute value of input
 */